docs/source/sctool/partials/sctool_repair.yaml

```diff
@@ -60,6 +60,8 @@ options:
     How many token ranges to repair in a single Scylla node at the same time.
     Zero (0) is a special value, the number of token ranges is adjusted to the maximum supported by node (see max_repair_ranges_in_parallel in Scylla logs).
     Changing the intensity impacts repair granularity if you need to resume it, the higher the value the more work on resume.
+    Setting intensity to a value greater than max supported results in trimming it internally.
```

Comment:
This sentence is confusing to me. It looks like it's about "max supported results". And I don't understand what "internally" means here. Perhaps it should go like this:

_If you set intensity to a value greater than the maximum supported by the node, intensity will be capped at that maximum._